terraform/modules/google-cloud/apps/relay/main.tf

```diff
@@ -121,6 +121,40 @@ resource "google_project_iam_member" "cloudtrace" {
   member = "serviceAccount:${google_service_account.application.email}"
 }
 
+# Create network
+resource "google_compute_network" "network" {
+  project = var.project_id
+  name    = "relays"
```

Comment:
Will probably need to change this to get it to deploy (i.e. trigger creation of a new network).